includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -744,51 +755,48 @@ public static function store_validation_errors( $validation_errors, $url, $args
 			$term_data = AMP_Validation_Error_Taxonomy::prepare_validation_error_taxonomy_term( $data );
 			$term_slug = $term_data['slug'];
 
-			if ( ! isset( $terms[ $term_slug ] ) ) {
```

Comment:
The only change here was removing the if condition wrapping this block of code. `$terms` will always be an empty array, and so there is need to check if `$terms[ $term_slug ]` is set.

Reply:
Ah yes you're right. Reverted the change in 64e47f343d3cf289de6ef3e0e0a9724a46bfc0be.